questions.md

```diff
@@ -0,0 +1,5 @@
+If I try to push to a repository to which I do not have a write permission, will it generate a pull request? 
```

Comment:
👋 Hi @joannakl ! When you push to a repository, it will not automatically generate a pull request, unless you have some sort of integration. If you were to push to a repository you do not have write access to, you will have to create a branch so that you can create a PR to get your changes incorporated into Master.